app/move/controllers/create/base.js

```diff
@@ -65,6 +65,21 @@ class CreateBaseController extends FormWizardController {
 
     next()
   }
+
+  saveValues(req, res, next) {
+    const { currentLocation } = req.session
+    const {
+      id: currentLocationId,
+      location_type: locationType,
+      can_upload_documents: canUploadDocuments,
+    } = currentLocation
```

Comment:
You can probably do the destructuring in one go -- is there a reason why there's an interim variable? (not that is very important at all!).

Reply:
Not sure what you mean? So just use `currentLocation.<property>` directly?

Reply:
Oh I see! `req.session.currentLocation`! Yep, thanks